packages/network-controller/src/create-network-client.ts

```diff
@@ -114,7 +102,11 @@ export function createNetworkClient(
 
   const provider = providerFromEngine(engine);
 
-  return { provider, blockTracker };
+  const destroy = () => {
+    blockTracker.destroy();
+  };
+
+  return { configuration: networkConfig, provider, blockTracker, destroy };
```

Comment:
Nit: It'd be nice to have tests for these two new properties (`configuration` and `destroy`)

Reply:
I'll try to take care of this tomorrow

Reply:
Tests added here: [ad08e04](https://github.com/MetaMask/core/pull/1439/commits/ad08e0499d95f18d0721942b9c2fe547774da15f)